relentless/core.py

```diff
@@ -440,6 +482,32 @@ def value(self, value):
             raise ValueError('Variable must be a float or int')
         self._value, self._state = self.clamp(float(value))
```

Comment:
I just noticed: we should not cast `value` to float here, it should just be as given.